### ISSUE TITLE
cheat engine cannot open wow 1.12.1 

### ISSUE THREAD

**Issue by gundamC** [sandboxie-plus/Sandboxie#3864]
### Describe what you noticed and did

![1](https://github.com/sandboxie-plus/Sandboxie/assets/38457592/8a2572e4-ae52-414c-bea5-3557a443d553)
![2](https://github.com/sandboxie-plus/Sandboxie/assets/38457592/8c9081bd-fdef-47b3-be55-40a1bdab432d)
when wow 1.12.1 running  in Sandboxie,cheat engine 7.5 cannot open it  .But when wow 1.12.1 running without Sandboxie,cheat engine 7.5 can open it.

### How often did you encounter it so far?

It has always been so

### Expected behavior

i just want to know why this happen.

### Affected program

Not relevant

### Download link

Not relevant.Can google wow 1.12.1 to find the download address, I forgot where to download it from.

### Where is the program located?

The program is installed only inside a sandbox (NOT in the real system anyway).

### Did the program or any related process close unexpectedly?

No, not at all.

### What version of Sandboxie are you running now?

Sandboxie-plus 1.13.3

### Is it a new installation of Sandboxie?

I recently did a new clean installation.

### In which sandbox type you have this problem?

In a standard isolation sandbox (yellow sandbox icon).

### Can you reproduce this problem on a new empty sandbox?

I can confirm it also on a new empty sandbox.

### What is your Windows edition and version?

windows 10 22H2

### In which Windows account you have this problem?

A local account (Administrator).

### Please mention any installed security software

no

### Did you previously enable some security policy settings outside Sandboxie?

In the default standard mode, I changed some settings after failed attempts, but all attempts failed.

### Sandboxie.ini configuration

```shell
Enabled=y
RecoverFolder=%{374DE290-123F-4565-9164-39C4925E467B}%
RecoverFolder=%Personal%
RecoverFolder=%Desktop%
BorderColor=#02f6f6,on,6
Template=OpenBluetooth
Template=SkipHook
Template=FileCopy
Template=qWave
Template=BlockPorts
Template=LingerPrograms
Template=AutoRecoverIgnore
ConfigLevel=10
UseFileDeleteV2=y
UseRegDeleteV2=y
AutoRecover=y
BoxNameTitle=-
SandboxieLogon=n
BlockNetParam=n
```

**Comment by love-code-yeyixiao** [comment]
From the image,we can know cheat engine is  running sandboxied.In the standrad sandbox,it does isoation based on tokens,so maybe the sandboxed process has no perminsion to access process virtual memory,can you run cheat engine non-sandboxed and try accessing wow process that is in a sandbox?

**Comment by gundamC** [comment]
yes,running cheat engine non-sandboxed can open WOW process which running sandboxied.Is there any way to get the cheat engine to open the World of Warcraft process in a sandbox environment?

**Comment by love-code-yeyixiao** [comment]
> yes,running cheat engine non-sandboxed can open WOW process which running sandboxied.Is there any way to get the cheat engine to open the World of Warcraft process in a sandbox environment?

Try closing safe isoation although you'll lose some security.
